lib/metatags/Metatags.js

```diff
@@ -1,73 +1,49 @@
 "use strict";
 
+var OpenGraphTag = require("./OpenGraphTag");
+var LinkTag = require("./LinkTag");
+var StandardTag = require("./StandardTag");
 var _ = require("underscore");
 
-var Metatags = function(pairs) {
-    if (!pairs) {
-        throw new Error("You cannot create empty metatags");
-    }
-
-    this.pairs = pairs;
-};
-
-Metatags.prototype = {
-
-    pairs: null,
+var Metatags = {
```

Comment:
can you make this object not a singleton. Like prevMetatags, you want to create instances of a group of metatags for the layout